x-pack/legacy/plugins/lens/public/types.ts

```diff
@@ -242,12 +265,37 @@ export interface LensMultiTable {
 
 export interface VisualizationLayerConfigProps<T = unknown> {
   layerId: string;
-  dragDropContext: DragContextState;
   frame: FramePublicAPI;
   state: T;
   setState: (newState: T) => void;
 }
 
+interface VisualizationDimensionConfig {
```

Comment:
Could we please use jsdoc style comments `/** */` here, so the comments will also show up in editor autocompletion/documentation for that code.

Reply:
Good suggestion in general, I will try to update this